src/copy.c

```diff
@@ -554,28 +563,26 @@ timescaledb_CopyGetAttnums(TupleDesc tupDesc, Relation rel, List *attnamelist)
 }
 
 static void
-copy_security_check(Relation rel, List *attnums)
+copy_constraints_and_check(ParseState *pstate, Relation rel, List *attnums)
```

Comment:
confused by the name of this function. Why isn't it just add_range_table_entry or something similar?

Reply:
It also checks permissions, which is the reason for the original name. No strong opinion on the name.